src/aero_rep_factory.F90

```diff
@@ -211,8 +211,8 @@ module camp_aero_rep_factory
 
   !> Identifiers for aerosol representations - used by binary
   !! packing/unpacking functions
-  integer(kind=i_kind), parameter :: AERO_REP_SINGLE_PARTICLE   = 1
-  integer(kind=i_kind), parameter :: AERO_REP_MODAL_BINNED_MASS = 2
+  integer(kind=i_kind), parameter :: AERO_REP_SINGLE_PARTICLE        = 1
+  integer(kind=i_kind), parameter :: AERO_REP_MODAL_BINNED_MASS      = 2
```

Comment:
Sure - white space somehow got added here in a file that you actually didn't appear to make any other modifications in. My suggestion should revert it back to how it was so that this file doesn't appear to be changed at all by your PR.